apps/volunteer/src/pages/shelters/profile/_components/ShelterRecruitments.tsx

```diff
@@ -1,53 +1,98 @@
 import { Box, Card, CardBody, Heading, Text } from '@chakra-ui/react';
+import { useSuspenseInfiniteQuery } from '@tanstack/react-query';
+import { useNavigate, useParams } from 'react-router-dom';
 import ApplicantStatus from 'shared/components/ApplicantStatus';
 import InfoSubtext from 'shared/components/InfoSubtext';
+import useIntersect from 'shared/hooks/useIntersection';
+import { createFormattedTime } from 'shared/utils/date';
+
+import {
+  getRecruitementsOfShelter,
+  RecruitmentOfShleter,
+} from '@/apis/shelter';
+
+function ShelterRecruitmentItem({
+  recruitmentId,
+  recruitmentTitle,
+  recruitmentStartTime,
+  recruitmentDeadline,
+  recruitmentApplicantCount,
+  recruitmentCapacity,
+}: RecruitmentOfShleter) {
+  const navigate = useNavigate();
+
+  const volunteerDate = createFormattedTime(
+    new Date(recruitmentStartTime),
+    'YY.MM.DD',
+  );
+
+  const deadline = createFormattedTime(
+    new Date(recruitmentDeadline),
+    'YY.MM.DD',
+  );
+
+  const goRecruitmentDetail = () => navigate(`/volunteers/${recruitmentId}`);
 
-const DUMMY_RECRUITITEM = {
-  title: '봉사자를 모집합니다.',
-  volunteerDay: '23.12.20',
-  deadline: '23.12.10',
-  applicant: 4,
-  capacity: 10,
-};
-
-type RecruitmentProps = {
-  title: string;
-  volunteerDay: string;
-  deadline: string;
-  applicant: number;
-  capacity: number;
-};
-
-function ShelterRecruitmentItem(recruitmentInfo: RecruitmentProps) {
-  const { title, volunteerDay, deadline, applicant, capacity } =
-    recruitmentInfo;
   return (
-    <Card p={4} pb={3.5} mb={2}>
+    <Card p={4} pb={3.5} mb={2} onClick={goRecruitmentDetail}>
       <CardBody pos="relative" p={0}>
         <Text pb={2} fontWeight={600}>
-          {title}
+          {recruitmentTitle}
         </Text>
-        <InfoSubtext title="봉사일" content={volunteerDay} />
+        <InfoSubtext title="봉사일" content={volunteerDate} />
         <InfoSubtext title="마감일" content={deadline} />
         <Box pos="absolute" right={0} bottom={0}>
-          <ApplicantStatus numerator={applicant} denominator={capacity} />
+          <ApplicantStatus
+            numerator={recruitmentApplicantCount}
+            denominator={recruitmentCapacity}
+          />
         </Box>
       </CardBody>
     </Card>
   );
 }
 
 export default function ShelterRecruitments() {
+  const { id } = useParams();
+  const shelterId = Number(id);
+
+  const {
+    data: { pages },
+    hasNextPage,
+    isFetchingNextPage,
+    fetchNextPage,
+  } = useSuspenseInfiniteQuery({
+    queryKey: ['shelter', shelterId, 'recruitments'],
+    queryFn: async ({ pageParam }) =>
+      (await getRecruitementsOfShelter(shelterId, pageParam, 10)).data,
```

Comment:
```suggestion
    queryFn: ({ pageParam }) =>
      getRecruitementsOfShelter(shelterId, pageParam, 10)
```

이렇게 해도 동작이 되지 않을까 생각됩니다! 🙋🏻‍♂️

Reply:
요청하신 코드로 작성을 하면 캐싱된 데이터에는 response.data 외의 response의 status, statusText, header, config,...등이 같이 저장되고 있습니다. 캐싱된 데이터에 데이터만 저장하고 싶어서 `.data`로 작성했는데 캐싱된 데이터에 response.data 외의 다른정보도 같이 저장하는 이유가 있을까요?